### ISSUE TITLE
No License, no source-code (does not belong on github?)

### ISSUE THREAD

**Issue by tamusjroyce** [shenyaocn/IP-Camera-Bridge#11]
Please include in the readme.md a link to the source code.

Otherwise, should this be hosted elsewhere? There is no license, no source code. And binary executables are checked into source control. Which is a big no-no?

**Comment by realgsong** [comment]
+1  What's the license for binary file? Is it possible for redistribution?

**Comment by xanda-escuyer** [comment]
Indeed. Of all the devices that require a transparent code base, a camera has to be near top of the list.
